packages/ui-core/src/components/Preloader/Preloader.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import './Preloader.less';
+import cnCreate from 'utils/cn';
+
+interface IPreloaderProps {
+    /** Color theme of preloader's circles */
+    color?: 'default' | 'black' | 'white';
+    /** Preloader size */
+    size?: 'small' | 'medium';
+    /** Custom class name */
+    className?: string;
+}
+
+const cn = cnCreate('mfui-preloader');
+class Preloader extends React.Component<IPreloaderProps> {
+    static propTypes = {
+        color: PropTypes.oneOf(['default', 'black', 'white']),
+        size: PropTypes.oneOf(['small', 'medium']),
+        className: PropTypes.string,
+    };
+
+    static defaultProps = {
```

Comment:
Было норм вроде
```suggestion
    static defaultProps: IPreloaderProps = {
```
Иначе в этот объект что угодно добавить можно